Update .gitignore remove old inclusions and improve readability

iJustLeyxo opened the pull request.

iJustLeyxo committed:
Remove old folder excludes from .gtiginore

HrGaertner commented:
@MineMuse changes will then cause a merge conflict.

HrGaertner reviewed (changes requested):
Keep .vscode in gitignore

iJustLeyxo commented:
Will do.

iJustLeyxo committed:
Add .vscode/ exclude to .gitignore and more

1) Add *.vscode/
2) Indentations for better readability
3) Remove unused inclusions

jon011235 commented on .gitignore:
```diff
@@ -1,39 +1,28 @@
-!*/
-!*.txt
+  *
+  !*/
```
What does this line do? Why would we want to include everything ending in a ','?

iJustLeyxo replied:
I do not quite understand what you mean about files ending in "," being included. Could you please provide further information like line numbers please?

Have added c and asm file extensions.

jon011235 reviewed (commented):
Maybe also add *.c files?
@DM8AT already used them sometimes and if we want to have a software part that would be the starting point

What do you think?

iJustLeyxo committed:
Add .asm and C-extensions to .gitignore
